libvirt/domain.go

```diff
@@ -252,6 +385,15 @@ func setCoreOSIgnition(d *schema.ResourceData, domainDef *libvirtxml.Domain) err
 	return nil
 }
 
+func updateCoreOSIgnition(d *schema.ResourceData, domainDef *libvirtxml.Domain, virConn *libvirt.Connect) error {
```

Comment:
I find the function naming strange. Is the difference between `updateCoreOSIgnition` and `setCoreOSIgnition` that former updates while latter sets? I don't think so and hence the naming is not great. `setCoreOSIgnition` should be named to contrast with the new `inlineUpdateCoreOSIgnition` function.

Reply:
I made it to be fwcfgUpdateCoreOSIgnition